docs/ConfigureScalarDL.md

```diff
@@ -2,126 +2,47 @@
-
-### Cosmos DB
-
-#### Configure schema-loading-custom-values
-
-To create a Scalar DL schema in Cosmos DB, you need to update the [schema-loading-custom-values.yaml](../conf/schema-loading-custom-values.yaml) file.
-
-**With Kubernetes secrets**
-
-If you have created the `scalardl` Kubernetes secret, you can use the following configuration.
-
-```yaml
-schemaLoading:
-  existingSecret: scalardl
-
-  database: cosmos
-  contactPoints: <Cosmos DB account endpoint>
-  cosmosBaseResourceUnit: 400
-```
-
-**Without Kubernetes secrets**
-
-If you have not created the `scalardl` Kubernetes secret, you can use the following configuration.
-
-```yaml
-schemaLoading:
-  database: cosmos
-  contactPoints: <Cosmos DB account endpoint>
-  password: <Cosmos DB account primary master key>
-  cosmosBaseResourceUnit: 400
-```
-
-#### Configure scalardl-custom-values 
-
-To deploy Scalar DL Ledger on Cosmos DB, you need to update the [scalardl-custom-values.yaml](../conf/scalardl-custom-values.yaml) file.
-
-**With Kubernetes secrets**
-
-If you have created the `scalardl` Kubernetes secret, you can use the following configuration.
 
-```yaml
-ledger:
-  existingSecret: scalardl
-  ....
-  scalarLedgerConfiguration:
-    dbContactPoints: <Cosmos DB account endpoint>
-    dbStorage: cosmos
-```
-**Without Kubernetes secrets**
+## Reference Table
 
-If you have not created the `scalardl` Kubernetes secret, you can use the following configuration.
+You can refer to the table below to get values to create Kubernetes Secrets and update the configuration files for Scalar DL deployment according to the database you chose.
 
-```yaml
-ledger:
-  ....
-  scalarLedgerConfiguration:
-    dbContactPoints: <Cosmos DB account endpoint>
-    dbPassword: <Cosmos DB account primary master key>
-    dbStorage: cosmos
-```
+| Database  | storageType | contactPoints              | username       | password                                 | dynamoBaseResourceUnit | cosmosBaseResourceUnit | 
+|:----------|-------------|----------------------------|----------------|------------------------------------------|------------------------|------------------------|
+| DynamoDB  | dynamo      | AWS_REGION                 | AWS_ACCESS_KEY | AWS_ACCESS_SECRET_KEY                    | 10                     | N/A                    |
+| Cosmos DB | cosmos      | Cosmos DB account endpoint | N/A            | Cosmos DB account primary/secondary key  | N/A                    | 400                    |
+| JDBC      | jdbc        | JDBC_CONNECTION_URL        | USERNAME       | PASSWORD                                 | N/A                    | N/A                    |
 
-To enable Auditor service, you can follow the [Enable Auditor](#enable-auditor) section.
+Note:- 
 
-#### Configure scalardl-audit-custom-values
+* JDBC denotes all relational databases supported by Scalar DL such as 
+  * MySQL, PostgreSQL, Oracle DB and SQL Server 
+  * AWS RDS for MySQL/PostgreSQL/Oracle/SQL Server
```

Comment:
Do we need to specify cloud-based databases separately?

Reply:
I just mentioned all the supported databases to avoid any confusion.
I will remove the cloud databases.